tests/DocumentationTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Str;
+use PHPUnit\Framework\TestCase;
+
+class DocumentationTest extends TestCase
+{
+    public function testIfDocumentationContainsAllPublicFunctions(): void
+    {
+        $functions = $this->getPublicFunctions();
+
+        foreach ($functions as $class => $methods) {
+            $trait = explode("\\", $class);
+            $fileName = Str::kebab(end($trait)) . ".html";
+            $filePath = "docs/elements/$fileName";
+
+            $replacements = [
+                "http-request.html" => "http.html",
+                "http-response.html" => "http.html",
+            ];
+
+            foreach ($replacements as $search => $replacement) {
+                if (str_ends_with($filePath, $search)) {
+                    $filePath = str_replace($search, $replacement, $filePath);
+                }
+            }
+
+            if (!file_exists($filePath)) {
+                $this->fail("Documentation file $filePath does not exist.");
+            }
+
+            $fileContent = file_get_contents($filePath);
+
+            if ($fileContent === false) {
+                $this->fail("Unable to read the file $filePath.");
+            }
+
+            foreach ($methods as $method) {
+                $containsFunction = str_contains($fileContent, $method) !== false;
+                $this->assertTrue($containsFunction, "Function $method is not documented in $filePath.");
+            }
+        }
+    }
+
+    protected function getBltTraits(): array
+    {
+        $classes = include "vendor/composer/autoload_classmap.php";
+        $bltTraits = [];
+
+        foreach (array_keys($classes) as $class) {
+            if (str_starts_with($class, "Blumilk\BLT\Features\Traits")) {
+                $bltTraits[] = $class;
+            }
+        }
+
+        return $bltTraits;
+    }
+
+    protected function getPublicFunctions(): array
+    {
+        $traitClasses = $this->getBltTraits();
```

Comment:
This is minor thing, but if `getPublicFunctions()` calls `getBltTraits()`, then the first one should be above second one.